crates/openapi/src/routes/merchant_account.rs

```diff
@@ -52,13 +52,12 @@ pub async fn merchant_account_create() {}
     post,
     path = "/v2/merchant_accounts",
     request_body(
-        content = MerchantAccountCreate,
+        content = MerchantAccountCreateWithoutOrgId,
```

Comment:
In documentation we shouldn't say this, it is just `MerchantAccountCreate` only

Reply:
@jarnura this won't be displayed in the openapi reference

Reply:
@jarnura Resolved. Please review.